tests/network/vmi_networking.go

```diff
@@ -287,7 +287,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 					// Create a virtual machine with an unsupported interface model
 					masqIface := libvmi.InterfaceDeviceWithMasqueradeBinding()
 					masqIface.Model = "gibberish"
-					customIfVMI := libvmi.NewAlpine(
+					customIfVMI := libvmi.NewCirros(
```

Comment:
The skip that is done in https://github.com/kubevirt/kubevirt/pull/7869 should be done here, I believe that now the test is cirros, it will fail without the skip.

Reply:
by here you mean this PR? or did you mean after this line?